### DISCUSSION TITLE
Bayer persona y Publico objetivo

### DISCUSSION THREAD

**Discussion by iguelAL** [Launch-X-Latam/MisionFrontEnd#160] [Q&A]
Hola, por lo que entendí de lo que dijo el Mission Comander, el Bayer Persona es el cliente ideal que contrataría el servicio de abogabot y el Público Objetivo es el público en general a quien le interesaria el servicio de abogabot, ¿Es cierto?

**Comment by doguedogue** [comment]
Aja, el BUYER Persona, es tu comprador, tu cliente, tu persona objetivo.
